WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyBarViewModel.cs

```diff
@@ -16,6 +16,9 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.Tiles.PrivacyRing;
 public partial class PrivacyBarViewModel : ViewModelBase
 {
 	private readonly SourceList<PrivacyBarItemViewModel> _itemsSourceList = new();
+	private const decimal GapBetweenSegments = 1.5m;
+	private const decimal EnlargeThreshold = 2m;
+	private const decimal EnlargeBy = 1m;
```

Comment:
CodeFactor is going to complain about this
```suggestion
	private const decimal GapBetweenSegments = 1.5m;
	private const decimal EnlargeThreshold = 2m;
	private const decimal EnlargeBy = 1m;
	private readonly SourceList<PrivacyBarItemViewModel> _itemsSourceList = new();
```